index.html

```diff
@@ -0,0 +1,95 @@
+            <p class="intro-1">"Hello! Sharing world"</p>
+            <h1>Be Happy, Let's fight for the peace throught Live Music Concert.</h1>
+            <p class="intro-2">A joyfull dayS to celebrate freedom trought music in the East the Republic Democratic of Congo. This activity help to reminde each others that a we can hope for a bright future.</p>
+            <h3>2023.02.23(THU)~25(SAT)</h3>
+            <p class="intro-3">@National Museum of Goma/North Kivu. Art Center and more.</p>
+        </section>
+
+        <section class="programs">
+            <h2>Main Program</h2>
+            <div class="line"></div>
+            <div class="program-container">
+                <div class="program">
+                    <div class="program-icon"><img src="image/lecture-conference-svgrepo-com.svg" alt="lecture"></div>
+                    <h3 class="program-title">Lecture</h3>
+                    <p>Listen to speakers from various tribe and province about the message of peace and love.</p>
+                </div>
+                <div class="program">
+                    <div class="program-icon"><img src="image/exhibition-museum-svgrepo-com.svg" alt="lecture"></div>
+                    <h3 class="program-title">Exhibition</h3>
+                    <p>Listen to speakers from various tribe and province about the message of peace and love.</p>
+                </div>
+                <div class="program">
+                    <div class="program-icon"><img src="image/comment-material-2-svgrepo-com.svg" alt="lecture"></div>
+                    <h3 class="program-title">Forum</h3>
+                    <p>Listen to speakers from various tribe and province about the message of peace and love.</p>
+                </div>
+                <div class="program">
+                    <div class="program-icon"><img src="image/workshop-svgrepo-com.svg" alt="lecture"></div>
+                    <h3 class="program-title">Workshop</h3>
+                    <p>Listen to speakers from various tribe and province about the message of peace and love.</p>
+                </div>
+                <div class="program">
+                    <div class="program-icon"><img src="image/light-svgrepo-com.svg" alt="lecture"></div>
+                    <h3 class="program-title">Ignite</h3>
+                    <p>Listen to speakers from various tribe and province about the message of peace and love.</p>
+                </div>
+            </div>
+
+            <button class="Program-btn" type="button">Join us to 2023 Festival Amani.</button>
+            <p class="view-program"><a href="#">SEE THE WHOLE PROGRAM</a></p>
+        </section>
+
+        <section class="features">
+            <h2>Featured Speakers</h2>
+            <div class="line"></div>
+            <div id="speaker-id"></div>
+
+            <button class="feature-btn" type="button">MORE <img src="image/down-arrow.svg" alt="down-arrow"></button>
+        </section>
+    </main>
```

Comment:
- [ ] Kindly follow the Behance design you are working on as strictly as possible. Currently, your design is missing two sections as seen in the original design which is the `Partner section` and the `Footer section` kindly implement these two missing sections in your project on the desktop view but optionally you can as well add them to your mobile view. Kindly work on this and you'll be sure you have replicated the design as much as possible.

Missing part 👇 

![beh01](https://user-images.githubusercontent.com/78634841/217487415-7526c695-6966-482c-95ca-af55d3235ad1.png)